pkg/apis/core/types.go

```diff
@@ -579,6 +605,20 @@ type PersistentVolumeClaimStatus struct {
 	// +featureGate=RecoverVolumeExpansionFailure
 	// +optional
 	ResizeStatus *PersistentVolumeClaimResizeStatus
+	// volumeAttributesClassName is the name of VolumeAttributesClass which is latest applied by
+	// the external-provisioner or external-resizer. This field is not set by default but when
+	// modification is complete, it's set same as VolumeAttributesClassName in spec.
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClassName
+	// +optional
+	VolumeAttributesClassName string
+	// modifyVolumeStatus stores status of modification operation.
+	// ModifyVolumeStatus is not set by default but when modification is complete, modifyVolumeStatus
+	// is set to empty string by the modify volume controller.
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClass
+	// +optional
+	ModifyVolumeStatus *VolumeAttributesClassStatus
```

Comment:
`VolumeAttributesClassName *string` is missing from PersistentVolumeSpec.

Reply:
thanks, I will fix it in next commit.

Reply:
@sunnylovestiramisu @xing-yang what do you think?

Reply:
Sounds good to me.

Reply:
cc @sunnylovestiramisu

Reply:
done in https://github.com/kubernetes/kubernetes/pull/118863/commits/0548cb2393898c205689dfa919a36b8687ef4f7d